example/hello_world.py

```diff
@@ -95,15 +106,15 @@ def attachments(test):
       test_version='1.0.0')
   test.AddOutputCallback(OutputToJSON(
       './%(dut_id)s.%(start_time_millis)s.json', indent=4))
-  test.AddOutputCallback(output.OutputToTestRunProto(
-      './%(dut_id)s.%(start_time_millis)s.json'))
+  #test.AddOutputCallback(output.OutputToTestRunProto(
```

Comment:
i would prefer the example by default not depend on the protos being there, but it's a handy reference to have there, which is why i left it in but commented out.